modules/nw-http2-haproxy.adoc

```diff
@@ -16,7 +16,7 @@ The connection from HAProxy to the application pod can use HTTP/2 only for re-en
 
 [WARNING]
 ====
-Using WebSockets with a re-encrypt route and with HTTP/2 enabled on an Ingress Controller requires WebSocket support over HTTP/2. WebSockets over HTTP/2 is a feature of HAProxy 2.6, which is unsupported in {product-title} at this time.
+Using WebSockets with a re-encrypt route and with HTTP/2 enabled on an Ingress Controller requires WebSocket support over HTTP/2. WebSockets over HTTP/2 is a feature of HAProxy 2.8, which is unsupported in {product-title} at this time.
```

Comment:
We need to remove this section. Websockets over HTTP/2 were first supported in haproxy-2.4. With OCP 4.14 we've been running haproxy 2.6, and now in OCP 4.16 the bump to haproxy 2.8 per this PR. When we drop this text we need to backport the doc changes to  OCP 4.15 and OCP 4.14.

Reply:
@frobware Thank you for your guidance! Updated. I will backport the removal for 4.15 and 4.14 in separate PRs.